lib/action_controller/responder.rb

```diff
@@ -233,7 +233,13 @@ def default_render
       if @default_response
         @default_response.call(options)
       else
-        controller.default_render(options)
+        if Rails.version >= "5"
```

Comment:
I think you can always pass a block. It simply won't do anything in earlier Rails versions. :)